src/bootstrap/bin/rustc.rs

```diff
@@ -217,6 +217,12 @@ fn main() {
         eprintln!("{prefix} libdir: {libdir:?}");
     }
 
+    if env::var_os("RUSTC_BOLT_LINK_FLAGS").is_some() {
+        if let Some("rustc_driver") = crate_name {
+            cmd.arg("-Clink-args=-Wl,-q");
+        }
+    }
```

Comment:
This was added *after* the debug printing, so the extra flags won't show up when the command is printed. Was that deliberate?

Reply:
Definitely not, good catch, thanks! https://github.com/rust-lang/rust/pull/123189